gateways/authorizenet/environment.go

```diff
@@ -0,0 +1,10 @@
+package authorizenet
+
+import "github.com/BoltApp/sleet/common"
+
+func authorizeNetHost(env common.Environment) string {
+	if env == common.Production {
+		return "https://api.authorize.net/xml/v1/request.api"
```

Comment:
I'm not too familiar with Auth.net's API, are all requests sent to this URL path? If the path is always the same let's change the client to store `url` instead of `host` and update this to be `authorizeNetURL`.

Reply:
makes sense and yeah it appears to be like this is used for all requests